server/migrations/versions/8b6b89bfbc97_backgroundtask.py

```diff
@@ -18,31 +18,7 @@
 depends_on = None
 
 
-def upgrade():
-    op.create_table(
-        "background_task",
-        sa.Column("created_at", sa.DateTime(), nullable=False),
-        sa.Column("updated_at", sa.DateTime(), nullable=False),
-        sa.Column("id", sa.String(length=200), nullable=False),
-        sa.Column("task_name", sa.String(length=200), nullable=False),
-        sa.Column("payload", sa.JSON(), nullable=False),
-        sa.Column("started_at", sa.DateTime(), nullable=True),
-        sa.Column("completed_at", sa.DateTime(), nullable=True),
-        sa.Column("error", sa.Text(), nullable=True),
-        sa.PrimaryKeyConstraint("id", name=op.f("background_task_pkey")),
-    )
-    op.add_column(
-        "round", sa.Column("draw_sample_task_id", sa.String(length=200), nullable=True)
-    )
-    op.create_foreign_key(
-        op.f("round_draw_sample_task_id_fkey"),
-        "round",
-        "background_task",
-        ["draw_sample_task_id"],
-        ["id"],
-        ondelete="cascade",
-    )
-
+def backfill():  # pragma: no cover
```

Comment:
I'm confused as to why this existing migration is changing. Is that expected?

Reply:
i was just doing a small refactor to increase code coverage, but looking at it again i may have made a small mistake

Reply:
just pushed a fix